### PR TITLE
🚸 get-exa: cross-chain, drawer & reminder fix

### PR DESCRIPTION
**Pull request by sebipap**
1. [🚸 get-exa: enable cross-chain](https://github.com/exactly/app/pull/1252/commits/be21a8d5d780537ffb558633a9cadbfbcef43dd6)

2. [🚸 MaturityDateReminder: dismissable + global state](https://github.com/exactly/app/pull/1252/commits/653fc5b9b91b4a0db12ed9bd7cb474bb8824acc7)

3. [🚸 get exa: move to drawer](https://github.com/exactly/app/pull/1252/commits/637ec729893c378e8fc50ea24ab3932977464920)

### CODE REVIEW

**Review by franm91 — APPROVED** [review]

File: utils/client.ts [hunk 1]
```diff
@@ -83,5 +83,6 @@ createWeb3Modal({
   chains,
   themeVariables: {
     '--w3m-font-family': 'Inter',
+    '--w3m-z-index': 1201,
```
Comment (sebipap) [inline utils/client.ts:86]: I used this value because `Mui.Drawer` `z-index` is 1200

File: pages/strategies.tsx [hunk 2]
```diff
@@ -231,16 +233,16 @@ const Strategies: NextPage = () => {
...
           ],
           button: (
-            <Link href={{ pathname: '/get-exa', query }} style={{ width: '100%' }}>
-              <Button fullWidth variant="contained">
+            <a style={{ width: '100%' }}>
+              <Button fullWidth variant="contained" onClick={openGetEXA}>
                 {t('Get EXA')}
               </Button>
-            </Link>
+            </a>
```
Comment (jgalat) [inline pages/strategies.tsx:240]: Remove the anchor, it is no longer necessary

File: components/VestingInput/index.tsx [hunk 3]
```diff
@@ -408,7 +409,12 @@ function VestingInput({ refetch }: Props) {
                 <Trans
                   i18nKey="Not enough EXA for reserve. <1>Get EXA</1>."
                   components={{
-                    1: <Link href="/get-exa" style={{ fontWeight: 700, textDecoration: 'underline' }} />,
+                    1: (
+                      <a
+                        onClick={openGetEXA}
+                        style={{ fontWeight: 700, textDecoration: 'underline', cursor: 'pointer' }}
+                      />
```
Comment (jgalat) [inline components/VestingInput/index.tsx:416]: Is it possible to make this an unstyled button? An anchor with no href acts as a plain div

File: pages/vesting.tsx [hunk 4]
```diff
@@ -110,7 +111,12 @@ const Vesting: NextPage = () => {
               <Trans
                 i18nKey="You can <1>get EXA</1> if you don’t have the required reserve amount."
                 components={{
-                  1: <Link href="/get-exa" style={{ textDecoration: 'underline' }} />,
+                  1: (
+                    <a
+                      onClick={openGetEXA}
+                      style={{ textDecoration: 'underline', fontWeight: 700, cursor: 'pointer' }}
+                    />
```
Comment (jgalat) [inline pages/vesting.tsx:118]: Same as above, can we make this a button?

**Review by franm91 — APPROVED** [review]

**Review by jgalat — APPROVED** [review]

**Pull request merged**